research/flamby/fed_isic2019/fenda_mkmmd/run_fold_experiment.slrm

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+#SBATCH --nodes=1
+#SBATCH --ntasks=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --cpus-per-task=8
+#SBATCH --gres=gpu:1
+#SBATCH --mem=32G
+#SBATCH --partition=a40
+#SBATCH --qos=m
+#SBATCH --job-name=fl_five_fold_exp
+#SBATCH --output=%j_%x.out
+#SBATCH --error=%j_%x.err
+#SBATCH --time=12:00:00
+
+###############################################
+# Usage:
+#
+# sbatch research/flamby/fed_isic2019/fenda_mkmmd/run_fold_experiment.slrm \
+#   path_to_config.yaml \
+#   path_to_folder_for_artifacts/ \
+#   path_to_folder_for_dataset/ \
+#   path_to_desired_venv/ \
+#   client_side_learning_rate_value \
+#   server_address
+#
+# Example:
+# sbatch research/flamby/fed_isic2019/fenda_mkmmd/run_fold_experiment.slrm \
+#   research/flamby/fed_isic2019/fenda_mkmmd/config.yaml \
+#   research/flamby/fed_isic2019/fenda/hp_results/ \
+#   /Users/david/Desktop/FLambyDatasets/fedisic2019/ \
+#   /h/demerson/vector_repositories/fl4health_env/ \
+#   0.0001 \
+#   0.0.0.0:8080
+#
+# Notes:
+# 1) The sbatch command above should be run from the top level directory of the repository.
+# 2) This example runs FENDA. As such the data paths and python launch commands are hardcoded. If you want to change
+# the example you run, you need to explicitly modify the code below.
+# 3) The logging directories need to ALREADY EXIST. The script does not create them.
+###############################################
+
+# Note:
+#	  ntasks: Total number of processes to use across world
+#	  ntasks-per-node: How many processes each node should create
+
+# Set NCCL options
+# export NCCL_DEBUG=INFO
+# NCCL backend to communicate between GPU workers is not provided in vector's cluster.
+# Disable this option in slurm.
+export NCCL_IB_DISABLE=1
+
+if [[ "${SLURM_JOB_PARTITION}" == "t4v2" ]] || \
+    [[ "${SLURM_JOB_PARTITION}" == "rtx6000" ]]; then
+    echo export NCCL_SOCKET_IFNAME=bond0 on "${SLURM_JOB_PARTITION}"
+    export NCCL_SOCKET_IFNAME=bond0
+fi
+
+# Process Inputs
+
+SERVER_CONFIG_PATH=$1
+ARTIFACT_DIR=$2
+DATASET_DIR=$3
+VENV_PATH=$4
+CLIENT_LR=$5
+SERVER_ADDRESS=$6
+MU=$7
+GAMMA=$8
+
+# Create the artifact directory
+mkdir "${ARTIFACT_DIR}"
+
+RUN_NAMES=( "Run1" )
+SEEDS=(2021 )
```

Comment:
As is, this script won't run 5 validations. I think we want to do that by default. So I think this is just an artifact of your experiments?

Reply:
oh no sorry this is probably accident.